docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -582,7 +582,7 @@ Host connection:
 .. include:: shared_snippets/SSH_password_prompt.txt
 
 ansible_connection
-    Connection type to the host. This can be the name of any Ansible connection plugin. SSH protocol types are ``smart``, ``ssh`` or ``paramiko``.  The default is smart. Non-SSH based types are described in the next section.
+    Connection type to the host. This can be the name of any Ansible connection plugin. SSH protocol types are ``ssh`` or ``paramiko``. The default is ssh. Non-SSH based types are described in the next section.
```

Comment:
```suggestion
    Connection type to the host. This can be the name of any Ansible connection plugin. SSH protocol types are ``ssh`` or ``paramiko``. The default is ``ssh``. Non-SSH based types are described in the next section.
```

Reply:
I wonder if we should remove that last sentence about "non-ssh based types"? It seems kind of arbitrary and useless to me.